bug_report.php

```diff
@@ -74,12 +75,29 @@
 require_api( 'string_api.php' );
 require_api( 'user_api.php' );
 require_api( 'utility_api.php' );
+require_api( 'api_token_api.php' );
 
-form_security_validate( 'bug_report' );
+# Check if this is a usual submission via UI or scripted autosubmission
```

Comment:
Here is idea behind - the auto reporting consists basically from one function: you post your report and get its status back. That's it. 

Do you want to search for SOAP support library for your language/IDE and include that complex code in your application just for that? 

If you could just make a single POST request (via OS functions) to report a bug and get its status back - wouldn't it be great?

So, it is not required, but would be a great option. For example, Fogbugz has this feature.